etna/analytics/mixins.py

```diff
@@ -10,8 +10,18 @@ class DataLayerMixin:
     datalayer.
     """
 
+    gtm_content_group: str = "Page"
+
```

Comment:
The `DataLayerMixin` mixin is not a `Page`. Why give it a value `Page` ?
How about `None` ?

Reply:
I think a blank string makes sense, as that way we're always dealing with a consistent data type

Reply:
Sure, go for it. Thanks.

Reply:
Actually, I went with `None` as the default, as it allows us to do some detection and raise an error when the value hasn't been set (or the method overridden)